SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/entity/Order.java

```diff
@@ -21,134 +21,134 @@
 import javax.persistence.Temporal;
 import javax.persistence.TemporalType;
 import javax.persistence.UniqueConstraint;
- 
+
 @Entity
-@Table(name = "Orders", //
-uniqueConstraints = { @UniqueConstraint(columnNames = {"Order_ID", "Product_ID", "Customer_ID"}) })
+@Table(name = "Orders") //
+//uniqueConstraints = { @UniqueConstraint(columnNames = {"Order_ID", "Product_ID", "Customer_ID"}) })
 public class Order implements Serializable {
- 
+
     /**
 	 * Maximal Constructor
 	 */
-	public Order(Date orderDate, String orderStatus, int amount, int prodId, Double prodRetailPrice,
-			int customerID, String customerName, String customerAddress, String customerEmail, String customerPhone,
+	public Order(Date orderDate, String orderStatus, int amount, Double prodRetailPrice,
```

Comment:
Correct - customer phone or any other customer-specific info should not be included in the Order entity's definition, as this would prevent our Order table from existing in 2NF. This is why I excluded it from the constructor, as it can be accessed directly through the Order's customer attribute.
So I guess I'm not seeing where the Java entity, Order, creates this? Since the green line indicates what now exists and the red indicates what I removed...